cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -67,7 +67,8 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
     testQuery(
       title = "Create a single-property index only if it does not already exist",
-      text = "If it is unknown if an index exists or not but we want to make sure it does, we add `IF NOT EXISTS`.",
+      text = "If it is unknown if an index exists or not but we want to make sure it does, we add `IF NOT EXISTS`. " +
+      "Note: The `IF [NOT] EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.",
```

Comment:
Since this is `CREATE` the `NOT` should not be optional
```suggestion
      "Note: The `IF NOT EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.",
```